### DISCUSSION TITLE
change default value of unsafe-disable-recovery in risedev.yml

### DISCUSSION THREAD

**Discussion by doki23** [risingwavelabs/risingwave#5560] [General]
How about changing the default value of `unsafe-disable-recovery` in 'risedev.yml' to true?
I met the problem that if I killed the cluster after creating a table, I couldn't start it up any more. (I configure components `etcd` and `minio`)
I'm not sure whether side effects will happen when use memory mode.

**Comment by skyzh** [comment]
RiseDev is for development purpose. Developers need to know what's going wrong instead of letting the system automatically recover.

**Reply by doki23** [reply]
Yes, you are right. I misunderstood this config.
